### ISSUE TITLE
Latest Docker build is stuck at version 1.8

### ISSUE THREAD

**Issue by alexgleith** [PDAL/PDAL#2401]
The below was run just now:

```
$ docker run --rm pdal/pdal pdal --version
--------------------------------------------------------------------------------
pdal 1.8.0 (git-version: ebc33b)
--------------------------------------------------------------------------------

$ docker pull pdal/pdal
Using default tag: latest
latest: Pulling from pdal/pdal
Digest: sha256:409aec41a953ea5a1af3643b0d5e60457de3ee0cd392c090992de39438fb90b9
Status: Image is up to date for pdal/pdal:latest
```

Basically, I think a build has picked up version 1.8, whereas is should be building on the latest. As it stands, there's no version 2.0 as a Docker image.

**Comment by abellgithub** [comment]
Version 1.8 is the latest released version.  If you're wanting version 1.8 with applied bug fixes, you can use pdal/alpine:1.8  If you're wanting a container that reflects the master branch, you can use pdal/alpine:latest

**Comment by alexgleith** [comment]
Well, what I'm wanting is a Docker image that will support Python3 and the pdal python library. I'm getting an issue when installing it, which I thought related to the PDAL Python library wanting version 2 of PDAL. I'll create another issue.
